NitroxServer/GameLogic/Players/PlayerData.cs

```diff
@@ -182,6 +182,33 @@ public Optional<NitroxId> GetSubRootId(string playerName)
             }
         }
 
+        /// <summary>
+        /// Searches in the persisted playerdata for a NitroxID associated to the given PlayerName
+        /// </summary>
+        /// <param name="playerName">the player to search for</param>
+        /// <returns></returns>
+        public NitroxId GetNitroxId(string playerName)
+        {
+            lock (playersByPlayerName)
+            {
+                PersistedPlayerData playerPersistedData = GetOrCreatePersistedPlayerData(playerName);
+                return playerPersistedData.PlayerNitroxId;
+            }
+        }
+
+        /// <summary>
+        /// Sets a NitroxID corresponding to a given PlayerName in persisted playerdata
+        /// </summary>
+        /// <param name="playerName">the playername to set the ID for</param>
+        /// <param name="id">the NitroxID to set</param>
+        public void SetPlayerNitroxID(string playerName, NitroxId id)
+        {
+            lock (playersByPlayerName)
+            {
+                playersByPlayerName[playerName].PlayerNitroxId = id;
```

Comment:
Is there a possibility for `playerName` to not be in `playersByPlayerName` (yet)? Given that this is called when the player is constructed and an ID was _not_ found in the save data?